### ISSUE TITLE
I've continuously got a problem with unicode

### ISSUE THREAD

**Issue by TylorShin** [oozcitak/xmlbuilder-js#154]
Hi.

I've seen 
```
Error: Invalid character () in string:  at index 0
```
error message for a month.
But I can't reproduce this.

I already turned on allowSurrogateChars to true.

So, What could be a problem?

**Comment by TylorShin** [comment]
The whold error message is below
```
Error: Invalid character () in string:  at index 0
  at XMLStringifier.module.exports.XMLStringifier.assertLegalChar(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:69108:16)
  at XMLStringifier.assertLegalChar(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:68959:60)
  at XMLStringifier.module.exports.XMLStringifier.eleText(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:68988:20)
  at new XMLText(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:35732:36)
  at XMLElement.module.exports.XMLNode.text(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:15301:16)
  at XMLElement.module.exports.XMLNode.txt(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:15501:20)
  at ? (/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:184852:28)
  at mapPlainText(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:184849:16)
  at ? (/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:184904:22)
  at ? (/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:6092:44)
  at List.__iterate(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:5599:14)
  at OrderedMap.__iterate(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:6091:26)
  at OrderedMap.forEach(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:7779:20)
  at Object.compose(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:184879:19)
  at EditorContainer.changeHandler(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:185824:41)
  at later(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:184579:32)
  at wrapped(/desktop_web/2017-04-25T09-37-30.795Z/bundleBrowser.js:143735:30)
```

**Comment by oozcitak** [comment]
Duplicate of #147. Should be fixed in 1f9b41a.
